core/pyproject.toml

```diff
@@ -16,12 +16,12 @@ classifiers = [
 python = "^3.9"
 pydantic = "^1.9.0"
 
-[tool.poetry.dev-dependencies]
-pytest = "^5.2"
-flake8 = "^3.9.2"
-mypy = "^0.910"
-isort = "^5.10.1"
-black = "^21.12b0"
+[tool.poetry.group.dev.dependencies]
+pytest = "*"
+flake8 = "*"
+mypy = "*"
+isort = "*"
+black = "*"
```

Comment:
Do you think it might be safer to be more specific with the version in the Action's lint yaml file:
https://github.com/calgo-lab/green-db/blob/main/.github/workflows/lint.yaml instead of being more lenient in our project?